hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -365,19 +365,35 @@ do
 done
 
 
+source hack/env.sh
+
 export ADMISSION_CONTROLLERS_ENABLED=true
 export ADMISSION_CONTROLLERS_CERTIFICATES_CERT_MANAGER_ENABLED=true
 export SKIP_VAR_SET=""
 export NAMESPACE="sriov-network-operator"
 export OPERATOR_NAMESPACE="sriov-network-operator"
 export CNI_BIN_PATH=/opt/cni/bin
 export OPERATOR_EXEC=kubectl
-export CLUSTER_TYPE=kubernetes
-export DEV_MODE=TRUE
 export CLUSTER_HAS_EMULATED_PF=TRUE
 
-echo "## deploy namespace"
-envsubst< $root/deploy/namespace.yaml | ${OPERATOR_EXEC} apply -f -
+
+HELM_VALUES_OPTS="\
+  --set images.operator=${SRIOV_NETWORK_OPERATOR_IMAGE} \
```

Comment:
hmm ci falling.

 maybe need to quote the images ? have a feeling its something with helm templating.

Reply:
We actually caught a regression introduced by
- https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/655

this is already paying back its effort